### PR TITLE
disallow lowercase import-as aliases

### PR DESCRIPTION
**Pull request by erik-krogh**
This was mostly due to me missing a case in the `getName()` predicate. 

Ignore the new `Acronyms should be PascalCase/camelCase` warnings, I'll fix them [over here](https://github.com/github/codeql/pull/8444).

### CODE REVIEW

File: javascript/ql/experimental/adaptivethreatmodeling/lib/experimental/adaptivethreatmodeling/AdaptiveThreatModeling.qll [hunk 1]
```diff
@@ -4,8 +4,8 @@
  * Provides information about the results of boosted queries for use in adaptive threat modeling (ATM).
  */
 
-private import javascript as raw
-private import raw::DataFlow as DataFlow
+private import javascript as JS
+private import JS::DataFlow as DataFlow
```
Comment (henrymercer) [inline javascript/ql/experimental/adaptivethreatmodeling/lib/experimental/adaptivethreatmodeling/AdaptiveThreatModeling.qll:8]:
```suggestion
private import javascript::DataFlow as DataFlow
```

**Review by henrymercer — DISMISSED** [review]
👍 on ML-powered queries

**Review by nickrolfe — DISMISSED** [review]
Ruby 👍

**Review by jbj — APPROVED** [review]
LGTM but also needs review from @github/codeql-csharp.

**Review by hvitved — APPROVED** [review]
C#/Ruby 👍

**Pull request merged**